apps/designer/app/designer/features/style-panel/controls/text/text-control.tsx

```diff
@@ -0,0 +1,228 @@
+import { ChangeEvent, FocusEvent, KeyboardEvent, useRef } from "react";
+import {
+  Box,
+  Text,
+  Grid,
+  IconButton,
+  TextField,
+  Tooltip,
+  IconButtonWithMenu,
+  numericScrubControl,
+} from "@webstudio-is/design-system";
+import { getFinalValue } from "../../shared/get-final-value";
+import { useIsFromCurrentBreakpoint } from "../../shared/use-is-from-current-breakpoint";
+import { ControlProps } from "../../style-sections";
+import { iconConfigs, StyleConfig } from "../../shared/configs";
+import { type StyleValue, type Unit, units } from "@webstudio-is/react-sdk";
+import { ChevronDownIcon } from "@webstudio-is/icons";
+
+const CSSGridLeft = {
```

Comment:
these shouldn't start from upper case, even in case of abbreviation its still cssGridLeft

Reply:
btw I used style suffix everywhere else: gridLeftStyle